tests/page-objects/infinite-scroller.ts

```diff
@@ -43,73 +45,54 @@ export class InfiniteScroller {
     this.firstItemTile = this.infiniteScrollerSectionContainer
       .locator('article')
       .first();
+
+    this.gridDisplayMode = this.displayStyleSelector.getByTestId('grid-button');
+    this.listDisplayMode = this.displayStyleSelector.getByTestId('list-detail-button');
+    this.compactDisplayMode = this.displayStyleSelector.getByTestId('list-compact-button')
   }
 
   async clickViewMode(viewModeLocator: LayoutViewModeLocator) {
     await this.displayStyleSelectorOptions.locator(viewModeLocator).click();
   }
 
-  async assertLayoutViewModeChange(viewMode: LayoutViewMode) {
-    switch (viewMode) {
-      case 'tile':
-        await expect(
-          this.displayStyleSelector.getByTestId('grid-button'),
-        ).toHaveClass('active');
-        expect(await expect(this.infiniteScroller).toHaveClass(/grid/));
-        return;
-      case 'list':
-        await expect(
-          this.displayStyleSelector.getByTestId('list-detail-button'),
-        ).toHaveClass('active');
-        expect(await expect(this.infiniteScroller).toHaveClass(/list-detail/));
-        return;
-      case 'compact':
-        await expect(
-          this.displayStyleSelector.getByTestId('list-compact-button'),
-        ).toHaveClass('active');
-        expect(await expect(this.infiniteScroller).toHaveClass(/list-compact/));
-        return;
-      default:
-        return;
-    }
+  async waitForFirstItemTile() {
+    const loadingTile = this.firstItemTile.locator('collection-browser-loading-tile');
+    await loadingTile.waitFor({ state: 'hidden' });
+    await this.firstItemTile.waitFor({ state: 'attached' });
+    await this.firstItemTile.waitFor({ state: 'visible' });
   }
 
   async hoverToFirstItem() {
-    expect(await this.firstItemTile.count()).toBe(1);
+    const tileHoverPane = this.firstItemTile.locator('tile-hover-pane');
 
-    await this.firstItemTile.hover();
-    await expect(this.firstItemTile.locator('tile-hover-pane')).toBeVisible({
-      timeout: 60000,
-    });
+    this.waitForFirstItemTile();
+    await this.firstItemTile.hover({ position: { x: 10, y: 20 } });
+    await this.firstItemTile.dispatchEvent('mouseover', { timeout: 5000 });
+    await tileHoverPane.waitFor({ state: 'attached' });
+    await tileHoverPane.waitFor({ state: 'visible' });
   }
 
-  async assertTileHoverPaneTitleIsSameWithItemTile() {
+  async tileHoverPaneAndItemTileText() {
     const textFirstItemTile = await this.firstItemTile
       .locator('#title > h4')
       .first()
       .innerText();
     const textTileHoverPane = await this.firstItemTile
       .locator('tile-hover-pane #title > a')
       .innerText();
-    expect(textFirstItemTile).toEqual(textTileHoverPane);
+    return textFirstItemTile === textTileHoverPane; 
   }
```

Comment:
Suggestion: Since the method performs essentially the same check as before, just without a formal assertion, I might expect a name more like `firstTileTitleMatchesHoverPaneTitle`.